src/DirectionTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+/**
+ * Adds horizontal / vertical functionality.
+ */
+trait DirectionTrait
+{
+    /** @var string Horizontal axis */
```

Comment:
```suggestion
    /** @var string */
```

(and on other places)